tests/wp-unit/include/Core/Metas/Modules/DeleteCommentMetaModuleTest.php

```diff
@@ -0,0 +1,588 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Metas\Modules;
+
+use BulkWP\Tests\WPCore\WPCoreUnitTestCase;
+
+/**
+ * Test Deletion of comment meta.
+ *
+ * Tests \BulkWP\BulkDelete\Core\Metas\Modules\DeleteUserMetaModule
+ *
+ * @since 6.0.0
+ */
+class DeleteCommentMetaModuleTest extends WPCoreUnitTestCase {
+
+	/**
+	 * The module that is getting tested.
+	 *
+	 * @var \BulkWP\BulkDelete\Core\Metas\Modules\DeleteCommentMetaModule
+	 */
+	protected $module;
+
+	/**
+	 * Setup the module.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->module = new DeleteCommentMetaModule();
+		$this->module->register( 'bulk-wp_page_bulk-delete-metas', 'bulk-delete-metas' );
+	}
+
+	/**
+	 * Add to test deleting comment meta from one comment.
+	 */
+	public function test_that_comment_meta_can_be_deleted_from_one_comment() {
+
+		$post_type  = 'post';
+		$meta_key   = 'test_key';
+		$meta_value = 'Test Value';
+
+		// Create a post.
+		$post = $this->factory->post->create( array( 'post_title' => 'Test Post' ) );
+
+		$comment_data = array(
+			'comment_post_ID' => $post,
+			'comment_content' => 'Test Comment',
+		);
+
+		// Create a comment.
+		$comment_id = $this->factory->comment->create( $comment_data );
+
+		add_comment_meta( $comment_id, $meta_key, $meta_value );
+
+		add_comment_meta( $comment_id, 'another', 'Another' );
```

Comment:
You are using the comment meta value `Another` here and then in `delete_comment_meta` you are checking for `another`.

Store the value in a variable and then use it.

This is happening in lot of test cases. Fix it in all the places.

Reply:
Fixed